README.md

```diff
@@ -51,3 +69,60 @@ using the following command:
 ```shell
 $ ENABLE_WEBHOOKS=false make run install
 ```
+
+## Testing the operator using Kind on Fedora 38
+
+This page covers running `kind` on Fedora 38 due to limitations on running it on RHEL - specifically in rootless mode.
```

Comment:
I'm curious, what is the issue on RHEL? I have no issues with running `kind` with rootless podman on macOS.

Reply:
Me and @johnbieren tried it. It is documented that you need to adjust one one kernel parameter on grub plus add another systemd set but they just make no effect (and IIRC update podman) but `kind` keeps failing to create cluster. I wasted some long hours trying to make it work - it could eventually work - but I think it does not worth the time. Creating a Fedora vm was more effective.

The error is this - it says that the steps I told you was not done. I started to debug, tried a lot of things, it but it feels a waste of time.
```
ERROR: failed to create cluster: running kind with rootless provider requires setting systemd property "Delegate=yes", see https://kind.sigs.k8s.io/docs/user/rootless/
```

Reply:
I see, thanks for the details. Maybe link to the documentation where the caveats for RHEL are described?

Reply:
the documentation makes it sound like you just run a few simple commands to make it work on RHEL, but as Leandro and I both experienced, this is not the case. Maybe the CSB install does something special that makes it not work well, no idea. I just know that I tried changing enough settings and it still didn't work so rather than continue to change system config stuff I don't fully understand, it was easier to just use a fedora vm

Reply:
It does not mention RHEL itself. It is just the fact that me and @johnbieren tried to apply the configs and it made no difference. Basically is this: https://kind.sigs.k8s.io/docs/user/rootless/
You might also search for "podman rootless"  then you will see dozen of ppl with the same complain - some about kind other about something else.